vhost-device-vsock/src/vhu_vsock_thread.rs

```diff
@@ -334,6 +390,9 @@ impl VhostUserVsockThread {
                 };
 
                 match stream {
+                    StreamType::Vsock(_) => {
+                        error!("Stream type should not be of type vsock");
```

Comment:
should we also do a `return` ?

Reply:
I remember doing a return here gives clippy error probably because the return is not necessary

Reply:
okay, I see